### PR TITLE
ATMOS Resin removes iciness from floors

### PR DESCRIPTION
**Pull request by Ergovisavi**
After a round of attempting to get rid of freon that some ruseman had dumped in the hallway and wanting to tear my goddamned hair out as an ATMOS tech:
* Atmos resin now makes floors unslippery when it's generated on a turf. Previously, while resin would eliminate the Freon from the air, it left the floor slippery and that made you need to pry up the floor, but then if you fucked up and accidentally let some freon loose it would make the base plating slippery which couldn't be fixed easily and it was just a whole lot of annoying hand swapping, etc
* Increased the amount of spread on the atmos resin launcher by 1, just to get a tad bit more coverage when the entire escape wing and every department is filled with gas

:cl:
tweak: Slightly increased the radius of the atmos resin launcher, and resin now makes floors unslippery
/:cl:

### CODE REVIEW

**Comment by Ergovisavi** [comment]
Why the hell was this tagged with removal

**Comment by optimumtact** [comment]
shit change, use a flamethrower to remove ice you pansy

**Comment by RandomMarine** [comment]
Or a space heater if you want to be a pansy.

**Comment by Iamgoofball** [comment]
the resin could use a buff

**Comment by Armhulen** [comment]
no????? last i checked it's foam that generates oxygen and filters out toxins (though i'd have to test that)

why does it also need to be an extremely easy counter to freon too

**Comment by ExcessiveUseOfCobblestone** [comment]
Resin is literally a 1 click wonder

Why/How is that fun exactly?

**Comment by Ergovisavi** [comment]
Because it actually gets ATMOS to get off their asses and do their jobs. There is a scale of "I should do this" and "Fuck it, this is stupid" and without carefully maintaining that, gameplay interaction ceases. Doing your job on a basic level should not require some kind of herculean effort on your part, nor should it require you flipping through 15 sets of tools and dragging an additional 3.

If you actually think attempting to clean up freon is an entertaining experience then I simply don't know what to say

**Comment by ExcessiveUseOfCobblestone** [comment]
I was under the impression you play atmos tech to like mess around with atmos and fix floods/breaches/etc.

I agree sometimes it seems like fixing one area takes the entire round length, but I don't think stripping the entire gameplay of fixing the breach with "click tile with item" is exactly the way to fix that, especially when it fixes EVERYTHING atmos related.

**Comment by Ergovisavi** [comment]
Resin doesn't fix holes in the floor, it just stems the bleed. It doesnt "fix" breaches, unless you count blocking off the entire area and condemning it to the void a valid solution. It filters and levels the temperature of the air, but does't reintroduce air. As useful as it is, there are still many holes in it.

ATMOS techs control the air on the station, and that includes using a filtering process that doesn't take ten years. It shouldn't be "LOL ROLLING FOR ANTAG FOR FLOODING".

**Comment by ExcessiveUseOfCobblestone** [comment]
So it Bandaids everything.

I can agree with the problem existing, and if this is what the backpack is suppose to do then I agree with the change, I just think it's making the job more casual.

**Comment by A-t48** [comment]
Last time I played, fixing floods was easy anyhow, though. Set vents to blow\suck as fast as possible, cycle out the entire atmosphere in the room. Done.

File: code/game/objects/effects/effect_system/effects_foam.dm [hunk 1]
```diff
@@ -250,6 +250,10 @@
 	. = ..()
 	if(isopenturf(loc))
 		var/turf/open/O = loc
+		O.wet = TURF_DRY
+		O.UpdateSlip()
+		if(O.wet_overlay)
+			O.cut_overlay(O.wet_overlay)
```
Comment (Cyberboss) [inline code/game/objects/effects/effect_system/effects_foam.dm]: There should be a proc on /turf/open to handle this, and if not, make one

**Review by Cyberboss — DISMISSED** [review]

**Comment by Cyberboss** [comment]
I personally think this is dumb. Atmos already has space heaters

**Comment by Iamgoofball** [comment]
i made freon and im okay with this, this is a much better way to handle it

**Comment by KorPhaeron** [comment]
Seems reasonable

**Comment by optimumtact** [comment]
Very well, make the change that @Cyberboss suggested and this will be merged.

**Comment by Ergovisavi** [comment]
Done @optimumtact

**Pull request merged**